src/runtime-rs/crates/resource/src/network/utils/link/create.rs

```diff
@@ -127,3 +127,52 @@ fn create_queue(name: &str, flags: libc::c_int) -> Result<(File, String)> {
     };
     Ok((file, req.get_name()?))
 }
+
+#[cfg(test)]
+mod tests {
+    use nix::unistd::getuid;
+    use rtnetlink::Handle;
+    use scopeguard::defer;
+
+    use crate::network::{network_pair::get_link_by_name, network_model::tc_filter_model::fetch_index};
+
+    use super::*;
+
+    async fn delete_link(handle: &Handle, name: &str) -> Result<(), rtnetlink::Error> {
```

Comment:
`delete_link` code should be reused (previously in `src/runtime-rs/crates/resource/src/network/network_pair.rs`) by save it in a common file and then import it.

Reply:
Fixed